css/store2.css

```diff
@@ -0,0 +1,254 @@
+main {
+    width: 100vw;
+    flex-direction: column;
+}
+
+
+/* Form Section */
+
+.form-side {
+    width: 100%;
+    box-sizing: border-box;
+    padding: 0 30px;
+}
+
+h2 {
+    color: #e0633a;
+    padding-top: 0;
+    font-size: 35px;
+}
+
+
+/* Container for the form rows */
+
+fieldset {
+    flex-direction: column;
+    border: none;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+legend {
+    margin: 10px 0;
+}
+
+
+/* Container for the label and input */
+
+.col {
+    flex-direction: column;
+    width: 100%;
+}
+
+.name {
+    width: 100%;
+    margin-right: 0%;
+}
+
+.name:first-of-type .half-width {
+    width: 100%;
+}
+
+.full-width {
+    width: 100%;
+}
+
+label {
+    width: fit-content;
+}
+
+input {
+    width: 100%;
+    padding: 7px;
+    margin-bottom: 20px;
+    border-radius: 5px;
+    border: 1px solid grey;
+    box-sizing: border-box;
+}
+
+
+/* City */
+
+select {
+    margin-bottom: 10px;
+    height: auto;
+    padding: 5px;
+    border: 1px solid grey;
+    border-radius: 5px;
+    width: 100%;
+}
+
+select:invalid {
+    color: grey;
+    font-size: 14px;
+    padding: 7px;
+}
+
+.city-wrapper {
+    flex-direction: column;
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    align-content: stretch;
+    align-items: flex-start;
+}
+
+.post {
+    width: 50%;
+}
+
+.post label {
+    margin-left: 0;
+    width: fit-content;
+}
+
+.postcode {
+    margin-left: 0;
+}
+
+.radio:last-of-type input[type="radio"] {
+    margin-left: 0;
+}
+
+
+/* Radio */
+
+input[type="radio"] {
+    appearance: none;
+    background-color: white;
+    margin: 1px 5px 30px 0;
+    color: grey;
+    width: 15px;
+    height: 15px;
+    border: 1px solid grey;
+    border-radius: 50%;
+    margin-right: 5px;
+    display: grid;
+    place-content: center;
+}
+
+input[type="radio"]::before {
+    content: "";
+    width: 7px;
+    height: 7px;
+    border-radius: 50%;
+    transform: scale(0);
+    transition: 120ms transform ease-in-out;
+    box-shadow: inset 0 0 5px 5px orange;
+}
+
+input[type="radio"]:checked::before {
+    transform: scale(1);
+}
+
+
+/* Checkbox */
+
+input[type="checkbox"] {
+    width: auto;
+}
+
+input[type="checkbox"]:before {
+    content: "";
+    display: flex;
+    width: 13px;
+    height: 13px;
+    margin-right: 5px;
+    background: white;
+    border: 2px solid grey;
+    align-items: center;
+    justify-content: center;
+}
+
+input[type="checkbox"]:checked:before {
+    background-image: url(../img/checkmark.svg);
+    background-size: contain;
+}
+
+.checkbox a {
+    color: #e0633a;
+    text-decoration: underline;
+}
+
+.checkbox {
+    font-size: 12px;
+    color: rgb(70, 70, 70);
+    margin-left: 5px;
+}
+
+.form-img {
+    background: url("../img/store-image_by-andrew-neel-unsplash.jpeg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    width: 100%;
+    background-position: right bottom;
```

Comment:
Because of this positioning the I could only see bottom half of the image, I would either increase the height of the box or use positioning to center the image.

Reply:
We were told that we shouldn't set both a height and width to an image so I set just the width and positioned it so the most important parts can be seen on smaller screens, background-position center didn't show a good portion of the image